wsclient/src/main/java/org/hpccsystems/ws/client/HPCCWsWorkUnitsClient.java

```diff
@@ -211,12 +230,19 @@ protected HPCCWsWorkUnitsClient(String protocol, String targetHost, String targe
     private void initWsWorkUnitsSoapProxy(String baseURL, String user, String pass)
     {
         wsWorkunitsServiceSoapProxy = new WsWorkunitsServiceSoapProxy(baseURL);
+        fallBackWorkunitsServiceSoapProxy = new org.hpccsystems.ws.client.gen.wsworkunits.v1_56.WsWorkunitsServiceSoapProxy(baseURL);
```

Comment:
can we delay this instantiation until after the if (wsWorkunitsServiceSoapProxy != null) conditional?